libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -89,17 +89,17 @@
         />
         <span
             class="fd-input-group__addon fd-input-group__addon--button"
+            [ngClass]="addOnClass"
```

Comment:
`[class]` is preferable over `[ngClass]`

Reply:
Hm, but not able to get the result with [class].